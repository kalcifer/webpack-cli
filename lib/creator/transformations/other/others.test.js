const defineTest = require('../../../transformations/defineTest');

defineTest(__dirname, 'amd', 'others-0', {amd : {
	jQuery: true,
	kQuery: false}
});
defineTest(__dirname, 'bail', 'others-0', {bail : true});
