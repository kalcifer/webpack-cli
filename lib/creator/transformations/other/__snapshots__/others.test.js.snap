// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`amd transforms correctly using "others-0" data 1`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    amd: {
        jQuery: true,
        kQuery: false
    }
}
"
`;

exports[`bail transforms correctly using "others-0" data 1`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    bail: true
}
"
`;
